BusinessTripApplication/Service/UserService.cs

```diff
@@ -113,5 +113,24 @@ public bool VerifyAccount(string id)
 
             return false;
         }
+
+        public User FindByEmail(User user)
+        {
+            IList<User> users;
+
+            try
+            {
+                users = userRepository.FindAll();
+            }
+            catch
+            {
+                throw;
+            }
+
+            User finded = users.Where(u => u.Email == user.Email).FirstOrDefault();
```

Comment:
This can be simplified by eliminating 'where' : 
`User finded = users.FirstOrDefault(u => u.Email == user.Email);`